src/style.css

```
@@ -1 +1,89 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Comfortaa:wght@300;400;700&family=Inter:wght@300;400;700;800&family=Roboto+Condensed:ital,wght@1,300;1,400;1,700&family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  margin: auto;
+  align-items: center;
+  max-width: 1200px;
+  height: 60px;
+}
+
+.link__logo {
+  outline: none;
+  text-decoration: none;
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  height: 100%;
+  margin-right: 50px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0;
+  height: 100%;
+}
+
+.nav__item {
+  padding: 0 10px;
+  cursor: pointer;
+}
+
+.nav__item:first-child {
+  padding-left: 0;
+}
+
+.nav__item:last-child {
+  padding-right: 0;
+}
+
+.nav__link {
+  outline: none;
+  text-decoration: none;
+  height: 100%;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14.06px;
+  color: #000;
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  border-bottom: 4px solid white;
+  border-radius: 4px;
+  padding-top: 4px;
+}
+
+.nav__link:active {
+  color: #00acdc;
+  border-radius: 1px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.hovered {
+  border-bottom: 4px solid #00acdc;
+}
```

you can set `border-bottom` for active link and remove class `hovered`
```suggestion
.is-active {
  color: #00acdc;
  border-bottom: 4px solid #00acdc;
}
```